docs/reference/api/api.md

```diff
@@ -5,7 +5,8 @@ The [APIs](/docs/development/introduction/glossary.html) in this document are or
 - [Platform](/docs/development/reference/platform.html): platform modules that provide consistent user experience.
 - [Drivers](/docs/development/reference/drivers.html): analog and digital input and outputs and digital interfaces.
 - [RTOS](/docs/development/reference/rtos.html): handling tasks and events in Mbed OS.
-- [Network Socket](/docs/development/reference/network-socket.html): network sockets, Ethernet, Wi-Fi, cellular and mesh networking.
+- [Socket API](socket-api.html): network socket API for TCP/IP.
```

Comment:
A flat socket-api.html link won't work. It has to have /docs/development etc.

Reply:
Why it does not work?
Documentation tells that it should work.
https://docs.internal.mbed.com/docs/master/publishing/text-styling-the-technical-details.html#linking